templates/nginx.j2

```diff
@@ -130,7 +130,7 @@ server {
 }
 
 server {
-    listen 80 default_server;
+    {% if not ansible_os_family == 'RedHat' %} listen 80 default_server; {% endif %}
```

Comment:
I did this change, because `nginx` on CentOS have other context `nginx.conf` file (probably on other RedHat based distros too) than on ubuntu and the diference is that on CentOS `nginx.conf` has `default_server` defined with `listen 80`, and file defined by `nginx.j2` have `default_server` defined with `listen 80` also, so this configuration produce error.
On the other hand ubuntu have `nginx.conf` without `default_server` so on ubuntu we have only one file with `default_server` and there installation doesn't produce error.

I hope that my explanation is clear enough